### PR TITLE
Slider fixes for small values

### PR DESCRIPTION
**Pull request by YonathanMeguira**
- moving the cursor with values < 1 is now smooth,
- updating the form now works with values < 1,
- slider gets updated when step < 1

### CODE REVIEW

**Comment by NetanelBasal** [comment]
Why do you have 111 commits?

File: src/app/components/slider/examples/slider-multi.ex.ts [hunk 1]
```diff
@@ -13,15 +13,16 @@ import { untilDestroyed } from 'ngx-take-until-destroy';
 })
 export class SliderMultiExampleComponent implements OnInit, OnDestroy {
   multiCursorForm = new FormGroup({
-    min: new FormControl(-50),
```
Comment (NetanelBasal) [inline src/app/components/slider/examples/slider-multi.ex.ts]: Don't override a simple multi example. Create a new example with the title "Decimal Values."

**Comment by NetanelBasal** [comment]
Please read the docs about our Commit Message Guidelines. http://dacorama.surge.sh/documentation/start

It should be 3 simple commits:
```ts
fix(slider): moving the cursor with values < 1 is now smooth. fixes #number
fix(slider): updating the form now works with values < 1. fixes #number
fix(slider):  slider gets updated when step < 1. fixes #number
```

File: src/app/components/slider/examples/slider-multi.ex.ts (hunk 1 above)
Comment (YonathanMeguira) [reply src/app/components/slider/examples/slider-multi.ex.ts]: done

File: src/app/components/slider/slider-docs.component.ts [hunk 2]
```diff
@@ -17,6 +19,8 @@ import SliderDisabledExampleCode from '!!raw-loader!./examples/slider-disabled.e
 
     <core-example title="Steps" [code]="SliderStepsExampleCode" [component]="SliderStepsExampleComponent"> </core-example>
 
+    <core-example title="Decimals" [code]="SliderDecimalExampleCode" [component]="SliderDecimalExampleComponent"> </core-example>
```
Comment (NetanelBasal) [inline src/app/components/slider/slider-docs.component.ts]: add input -> title="Decimal"

Comment (YonathanMeguira) [reply src/app/components/slider/slider-docs.component.ts]: this input is already there, not sure what you meant

Comment (NetanelBasal) [reply src/app/components/slider/slider-docs.component.ts]: Sorry, didn't notice. What about the 110 commits? Why is that?

Comment (YonathanMeguira) [reply src/app/components/slider/slider-docs.component.ts]: I honestly don't know, it acts as if the slider was never merged to master. Maybe this is due to the 'migration' of the new docs ?

**Pull request closed without merging**